docs/preflight/v1beta3-overview.md

```diff
@@ -0,0 +1,505 @@
+---
+title: "Preflight v1beta3: Templated Checks"
```

Comment:
@paigecalvert what do you think of the information architecture here for the sidebar and how these three articles fit in with everything else?